packages/block-editor/src/utils/transform-styles/index.js

```diff
@@ -31,11 +26,7 @@ const transformStyles = ( styles, wrapperClassName = '' ) => {
 		if ( baseURL ) {
 			transforms.push( urlRewrite( baseURL ) );
 		}
-		if ( transforms.length ) {
-			return traverse( css, compose( transforms ) );
-		}
-
-		return css;
+		return traverse( css, compose( transforms ) );
```

Comment:
This shouldn't use compose since the result of the second transform gets called with the result of the first, it should just be fed the original arguments, eg:

```js
return traverse( css, ( ...args ) => {
  transforms.map( ( transform ) => transform( ...args ) );
} );
```

But that would change the old `traverse` api